labelbox/schema/data_row_metadata.py

```diff
@@ -664,11 +686,33 @@ def _upsert_schema(
         self.refresh_ontology()
         return _parse_metadata_schema(res)
 
+    def _find_option_id_by_name(self, schema_name: str, option_name: str) -> DataRowMetadataSchema:
+        # Precondition: already validated that schema_name is a valid Schema name
+        try:
+            return self._fields_by_name[schema_name][option_name].uid
```

Comment:
this one definitely should use `self.get_by_name(name)[option].uid`

Reply:
Good catch! I've updated it.